pubspec.yaml

```diff
@@ -13,7 +13,10 @@ dependencies:
   barback: '>=0.14.2 <0.16.0'
   logging: '>=0.9.0 <0.12.0'
   path: '>=0.9.0 <2.0.0'
-  smoke: '>=0.1.0 <0.4.0'
+  #smoke: '>=0.1.0 <0.4.0'
+  test: any
```

Comment:
also would be good to list a minimum version, rather than "any". e.g. `^0.12.4+9`